packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -45,28 +51,65 @@ interface CreateSandboxProps {
   isModal?: boolean;
 }
 
+type EssentialsState =
+  | {
+      state: 'loading';
+    }
+  | {
+      state: 'success';
+      essentials: ITemplateInfo[];
+    }
+  | {
+      state: 'error';
+      error: string;
+    };
+
 export const CreateSandbox: React.FC<CreateSandboxProps> = ({
   collectionId,
   initialTab,
   isModal,
 }) => {
-  const { hasLogIn } = useAppState();
+  const { hasLogIn, activeTeamInfo, user } = useAppState();
   const actions = useActions();
 
+  const isRepositoriesPage = location.pathname.includes('/repositories');
+  const isDashboardPage = location.pathname.includes('/dashboard');
+  const isUser = user?.username === activeTeamInfo?.name;
+
+  const [essentialState, setEssentialState] = useState<EssentialsState>({
+    state: 'loading',
+  });
+
   const tab = useTabState({
     orientation: 'vertical',
-    selectedId: initialTab || 'create',
+    selectedId: initialTab || isRepositoriesPage ? 'import' : 'create',
```

Comment:
is this checking if you are on `/dashboard/repositories`, as in the new page where we show the v2 repos?

Reply:
Nope, it check if the user is on the `codesandbox.io/dashboard/repositories`, not the v2 version. I guess we should rename this to "synced sandboxes" page and update the url as well? What is the latest update / name for that?